website/blog/2018_week_52/Gardener_Cookies.md

```diff
@@ -1,29 +1,27 @@
 ---
 title: Gardener Cookies
-level: intermediate
-category: Operation
-scope: app-developer
+publishdate: 2018-12-25
 ---
 
 # Green Tea Matcha Cookies
 
 For a team event during the Christmas season we decided to completely reinterpret the topic `cookies`. :-)
 
-<img style="width: 50%; height: auto; margin: 0, auto" src="slider/cookie-00.jpg"/>
```

Comment:
Why are we replacing the `<img` tags with markdown?
markdown syntax does not allow styles like `width: 50%`